src/constants.ts

```diff
@@ -0,0 +1,13 @@
+import { SupportedNetworks } from './types'
+
+export const AUTOMATE_MD5_ADDRESS = '0x00000000e7fdc80c0728d856260f92fde10af019' // md5 hash of the word 'automate'
```

Comment:
should we declare type for these constants?

Reply:
In the case of primitives like inline string there's no need to additionally declare the type - VSC will know (and so will TS compiler). When you hover over a variable it will tell you what type it thinks a variable is. If it says `any` then you need to declare a type (unless you actually want `any`). Otherwise you don't need to do it most of the time, unless you want to force some type.
More on that here: https://www.typescriptlang.org/docs/handbook/type-inference.html